src/qumulo/azext_qumulo/azext_metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+    "azext.isPreview": true,
```

Comment:
```suggestion
```

Reply:
@Jing-song, as it will be released as GA, we need remove preview tag here. I updated it for you.